lib/fs.js

```diff
@@ -1112,14 +1114,14 @@ function writeAll(fd, buffer, offset, length, position, callback) {
 }
 
 fs.writeFile = function(path, data, options, callback) {
-  var callback = maybeCallback(arguments[arguments.length - 1]);
+  callback = maybeCallback(arguments[arguments.length - 1]);
```

Comment:
even if writing to `/dev/null` I doubt it could be run enough to notice a difference. and from the looks of it, nothing that way was actually changed. Just removed the unnecessary `var`.

@petkaantonov re that deopt example. that true even if `'use strict'` is used?